Calculator/ViewController.swift

```diff
@@ -73,110 +59,115 @@ class ViewController: UIViewController {
-        let currentActual = auxFormatter.string(from: NSNumber(value: actual))!
-        if !operating && currentActual.count >= maxLenght{
+        let currentActual = auxFormatter.string(from: NSNumber(value: currentHandler))!
+        if !operating && currentActual.count >= maxLenght {
             return
         }
         resultLabel.text = resultLabel.text! + (kdecimalSeparator ?? "")
         decimal = true
     }
+
     @IBAction func numberAction(_ sender: UIButton) {
-        
         btnRestart.setTitle("C", for: .normal)
-        
-        var currentActual = auxFormatter.string(from: NSNumber(value: actual))!
-        if !operating && currentActual.count >= maxLenght{
+
+        var currentActual = auxFormatter.string(from: NSNumber(value: currentHandler))!
+        if !operating && currentActual.count >= maxLenght {
             return
         }
-        
-        if operating{
-            result = result == 0 ? actual : result
+
+        if operating {
+            result = result == 0 ? currentHandler : result
             resultLabel.text = ""
             currentActual = ""
             operating = false
         }
-        
+
         if decimal {
             currentActual = "\(currentActual)\(kdecimalSeparator ?? "")"
             decimal = false
         }
-        
+
         let number = sender.tag
-        actual = Double(currentActual + String(number))!
-        resultLabel.text = printFormatter.string(from: NSNumber(value: actual))
-        print(sender.tag)
+        currentHandler = Double(currentActual + String(number))!
+        resultLabel.text = printFormatter.string(from: NSNumber(value: currentHandler))
     }
-    
-    //Limpiar valores
-    func clear(){
+
+    // Clean values
+    func clear() {
         operation = .none
         btnRestart.setTitle("AC", for: .normal)
-        if actual != 0 {
-            actual = 0
+        if currentHandler != 0 {
+            currentHandler = 0
             resultLabel.text = "0"
-        }else{
+        } else {
             result = 0
-            resultado()
+            currentResult()
         }
     }
     
-    func resultado(){
-        switch operation{
+    //Cases operators
+    func currentResult() {
+        switch operation {
         case .none:
-            //no hara nada
+            // no hara nada
             break
         case .sum:
-            result = result + actual
+            result = result + currentHandler
             break
         case .res:
-            result = result - actual
+            result = result - currentHandler
             break
         case .mult:
-            result = result * actual
+            result = result * currentHandler
             break
         case .div:
-            result = result / actual
+            result = result / currentHandler
             break
+        case .percent:
+            currentHandler = currentHandler / 100
+            result = currentHandler
```

Comment:
Could be nice to separate all the different cases in different functions in order to have the logic of the calculation separated